tests/util/localregistry/localregistry.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 100Mi
+            memory: 5000Mi
```

Comment:
I think this pr has these changes right? https://github.com/istio/istio/pull/5838/files/a729599a3160406178ad514c4a0ca078063c0e13..eb99de178214941e9bbb19966c402ac05bdfcc3c